nifi-nar-bundles/nifi-aws-bundle/nifi-aws-abstract-processors/src/main/java/org/apache/nifi/processors/aws/regions/AWSRegions.java

```diff
@@ -18,7 +18,8 @@
 
 public enum AWSRegions {
 
-    GovCloud("us-gov-west-1", "AWS GovCloud (US)"),
+    GovCloud("us-gov-west-1", "AWS GovCloud West (US)"),

```

Comment:
In the past, the display names have always been kept as what AWS shows. 
AWS docs still show:
 - AWS GovCloud (US-East)
 - AWS GovCloud (US)

Maybe it would be better to revert the change to `AWS GovCloud West (US)`? Then it will match the docs.

Reply:
I'm going to correct myself. Some docs say (US). Some say, `(US-West)` and `(US-East)`.

https://docs.aws.amazon.com/govcloud-us/index.html#lang/en_us

So maybe that would be better?

Reply:
I can change the display name -- what about the enum itself though? Changing the enum would break any existing flows I think? Just leave it so that GovCloud is implicitly West?

Reply:
Just the display name is all I'm worried about, everything else looks fine.